src/main/java/pl/marcin/project/configuration/TomTomConfiguration.java

```diff
@@ -0,0 +1,29 @@
+package pl.marcin.project.configuration;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.reactive.function.client.WebClient;
+
+@Configuration
+public class TomTomConfiguration {
+    private final String baseUrl = "https://api.tomtom.com";
+
+    @Bean
+    public WebClient.Builder webClientBuilder() {
+        return WebClient.builder();
+    }
+
+    @Bean
+    public WebClient webClient(WebClient.Builder webClientBuilder) {
+        return webClientBuilder.baseUrl(baseUrl).build();
+    }
+
+    @Bean
+    ObjectMapper objectMapper() {
+        ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
instead of having multiple `ObjectMappers` in one line, we could use `var` instead of a type  - we know from the variable's name AND its constructor what it will be